examples/step-36/step-36.cc

```diff
@@ -276,17 +271,21 @@ namespace Step36
           for (unsigned int i = 0; i < dofs_per_cell; ++i)
             for (unsigned int j = 0; j < dofs_per_cell; ++j)
               {
-                cell_stiffness_matrix(i, j) +=
-                  (fe_values.shape_grad(i, q_point) *
-                     fe_values.shape_grad(j, q_point) +
-                   potential_values[q_point] *
-                     fe_values.shape_value(i, q_point) *
-                     fe_values.shape_value(j, q_point)) *
-                  fe_values.JxW(q_point);
-
-                cell_mass_matrix(i, j) += (fe_values.shape_value(i, q_point) *
-                                           fe_values.shape_value(j, q_point)) *
-                                          fe_values.JxW(q_point);
+                cell_stiffness_matrix(i, j) +=           //
+                  (fe_values.shape_grad(i, q_point) *    //
+                     fe_values.shape_grad(j, q_point)    //
```

Comment:
I am a bit surprised that clang-format wants to add two spaces here but not in the mass matrix. Would surrounding the contraction with parentheses fix it?

I am not overly concerned, just slightly annoyed at clang-format's apparent inconsistency (which I suppose is due to the `+` sign on the next line).

Reply:
Yes it is due to the + sign.